### PR TITLE
making it work

### PR DESCRIPTION
**Pull request by vassjozsef**

### CODE REVIEW

File: VideoQuickstart/Web.config.example [hunk 1]
```diff
@@ -4,14 +4,14 @@
   http://go.microsoft.com/fwlink/?LinkId=301880
   -->
 <configuration>
-  <appSettings file="Local.config">
+  <appSettings">
```
Comment (dprothero) [inline VideoQuickstart/Web.config.example]: Left a stray double-quote character there.

**Comment by dprothero** [comment]
What didn't work about the Local.config setup?

File: VideoQuickstart/Web.config.example (hunk 1 above)
Comment (vassjozsef) [inline VideoQuickstart/Web.config.example]: Fixed.

**Comment by vassjozsef** [comment]
First of all, it was missing. Then I checked https://github.com/TwilioDevEd/sync-quickstart-csharp and that does not have `Local.config` either.

**Comment by dprothero** [comment]
That was the intended behavior, though it looks like the readme needs to make that more clear. Your change will work, too, however.

**Comment by vassjozsef** [comment]
I tested on a Windows box with Visual Studio Community Edition and worked.

**Pull request merged**